site/pages/core/quickstart.mdx

```diff
@@ -49,6 +49,49 @@ this package use.
 // [!include ~/shared/core/config.ts]
 ```
 
+## Watch and Hydrate
+
+The next step is to watch for any changes in the client state, and hydrate the account state. This is crucial for ensuring that the signer is properly defined and ready.
+
+1. Watch the client: You need to watch the smart account client to identify changes that could affect the signer and client state.
+2. Hydrate the account state: After detecting state changes, use the hydrate function on your config.
```

Comment:
This description makes it sound like you should call hydrate every time the state changes, but the code snippet below calls it one time only and not in response to a state change. Also the snippet below calls the `onMount` function returned from `hydrate`, but it's not clear to me what that does and when I should call it. It would help me if there were more of a description of what these functions do and when I should call them.

(I don't know the answer myself. This is from the perspective of someone unfamiliar with these functions.)